packages/wonder-stuff-ci/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "engines": {
+        "node": ">=16"
+    },
+    "name": "@khanacademy/wonder-stuff-ci",
+    "version": "0.0.1",
+    "description": "Functions for creating automation and scripts.",
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
```

Comment:
I was trying to get the test to run with `yarn jest`, but that didn't work because of typing. But using the yarn command it did work!

Reply:
Ah yeah - I think this handles some of the trickiness with being in a monorepo, if I remember correctly.

Reply:
Typically, you have to run `yarn jest` and `yarn test` from the root directory. This confusing command at the package level basically does that for you, so you can do `yarn test` in one of the package folders, and it will do the right thing.